internal/mempool/v1/reactor.go

```diff
@@ -187,7 +194,8 @@ func (r *Reactor) handleMempoolMessage(envelope p2p.Envelope) error {
 func (r *Reactor) handleMessage(chID p2p.ChannelID, envelope p2p.Envelope) (err error) {
 	defer func() {
 		if e := recover(); e != nil {
-			err = fmt.Errorf("panic in processing message: %v", e)
+			r.panicHandler(e)
```

Comment:
The default changes the semantics, doesn't it? Previously the handler swallowed the panic—here the default re-panics. I think what we probably want is just an empty function, e.g.,

```go
logPanic: func(interface{}) {},
```

or words to that effect.

Reply:
Great catch, thank you. The conversation about panic'ing in a panic from slack must have become lodged in my brain. Fixed.